cloudbuild.yaml

```diff
@@ -0,0 +1,19 @@
+steps:
+- name: golang:1.12
+  env:
+  - CGO_ENABLED=0
+  - GOOS=linux
+  args:
+  - go
+  - build
+  - -a
+  - -installsuffix
+  - cgo
+  - -o
+  - policy_asserter
+  - .
+- name: 'gcr.io/cloud-builders/docker'
+  args: [ 'build', '-t', 'gcr.io/$PROJECT_ID/sslpolicy-asserter:1.2', '-t', 'gcr.io/$PROJECT_ID/sslpolicy-asserter:latest', '.' ]
+images:
```

Comment:
Since this is a public github repo: where are the Docker images stored?

Reply:
Please see my comments at https://github.com/commercetools/gcp-ssl-policy-asserter/pull/3#discussion_r413465458